JitsiConnection.js

```diff
@@ -23,11 +23,13 @@ export default function JitsiConnection(appID, token, options) {
     this.options = options;
     this.xmpp = new XMPP(options, token);
 
+    /* eslint-disable max-params */
     this.addEventListener(JitsiConnectionEvents.CONNECTION_FAILED,
-        (errType, msg) => {
+        (errType, msg, credentials, details) => {
```

Comment:
We explicitly disable the max-params rule in order to add an extra parameter which is always "undefined" and always ignored :)

Reply:
Ah, I see now that it is used elsewhere.